problems/objects.js

```diff
@@ -235,7 +368,14 @@ const updateList = () => {};
   * @param {Object} obj
   * @returns {number} Number of properties.
   */
- const propertyCount = () => {};
+ const propertyCount = (obj) => {
+   let key = Object.keys(obj)
```

Comment:
You can get the number of properties by using .length()
Try refactoring this code to be one line.